src/components/NotFoundPanel.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <app-panel class="not-found-panel">
```

Comment:
Can this panel also be reused in pages/error/query ?

Reply:
I'll give it a try

Reply:
So what is the outcome?

Reply:
I think it worked :+1: